make_licenses.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Comcast Corporation
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# 
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
You should use `set -e` so that in case e.g. dbmaker fails to build it doesn't try to use the binary anyway.

Reply:
Yeah, this whole file needs better error checking. In general, though, errors here will cascade to build failures later, so it'll eventually be caught, even if not reported as easily as one might hope. I'll make a note.